docs/faq.md

```diff
@@ -200,8 +200,5 @@ Discord is your answer: join and talk to us:
 
 [![Discord](https://img.shields.io/badge/Discord-GDB--GEF-yellow)](https://discordapp.com/channels/705160148813086841/705160148813086843) [[invite link](https://discord.gg/HCS8Hg7)]
 
-If you cannot find the answer to your issue in the FAQ (here!), the [Issues](https://github.com/hugsy/issues) then feel free to drop by and ask away!
-
-
-
+If you cannot find the answer to your issue in the FAQ (here!), the [Issues](https://github.com/hugsy/gef/issues) then feel free to drop by and ask away!
```

Comment:
This sentence doesn't make sense. Missing an 'or'? e.g. 'cannot find in the faq or the issues then feel...'